mackerel-plugin-mysql/lib/mysql.go

```diff
@@ -64,8 +64,9 @@ func (m MySQLPlugin) defaultGraphdef() map[string]mp.Graphs {
 			Metrics: []mp.Metrics{
 				{Name: "Select_full_join", Label: "Select Full JOIN", Diff: true, Stacked: false},
 				{Name: "Select_full_range_join", Label: "Select Full Range JOIN", Diff: true, Stacked: false},
+				{Name: "Select_range", Label: "Select Range", Diff: true, Stacked: false},
+				{Name: "Select_range_check", Label: "Select Range Check", Diff: true, Stacked: false},
 				{Name: "Select_scan", Label: "Select SCAN", Diff: true, Stacked: false},
-				{Name: "Sort_scan", Label: "Sort SCAN", Diff: true, Stacked: false},
```

Comment:
I think it is typo because `Sort_scan` is appeared in sort graph
https://github.com/mackerelio/mackerel-agent-plugins/blob/2ba382fdfe71ec75b144434c8e2379a9ece9ae2f/mackerel-plugin-mysql/lib/mysql.go#L581

Reply:
fixed in 5236c0e